docs/api.md

```diff
@@ -1,109 +1,44 @@
-# API
+# API 
 
-Please have a look at the documentation and examples of the interactive Swagger UI: [https://oqt.ohsome.org/api/docs](https://oqt.ohsome.org/api/docs)
+Please have a look at the request examples of the [interactive Swagger UI](https://oqt.ohsome.org/api/docs).
+Using the Swagger UI one can create valid `CURL` requests with using an interactive interface.
 
-## A note on the difference of `GET` and `POST` requests to the `indicator` or `report` endpoints
 
-To the endpoints `/indicator` and `/report` a request can be made using either the `GET` or `POST` method.
-For `GET` requests to either of those endpoints only the parameters `dataset` and `feature_id` are supported, not the `bpolys` parameter.
-That means that it is not possible to create an indicator or a report for a custom AOI using the `bpolys` parameter by making a `GET` request to the API.
+## Indicator Endpoint
 
-Using the `POST` method all parameters are supported.
+Three types of requests can be made:
 
+- To request an Indicator for an AOI defined by OQT provide following parameters: `name`, `layerName`, `dataset` and `featureId`
+- To request an Indicator for a custom AOI provide following parameters: parameters: `name`, `layerName` and `bpolys`
+- To request an Indicator for a custom AOI and custom data provide following parameters: `name`, `bpolys` and `layer`
 
-## Request examples using Bash and `curl` command
+Depending on the input, the output will be a GeoJSON Feature or FeatureCollection with the results of the Indicator as properties. The Feature properties of the input GeoJSON will be preserved if they do not collide with the properties set by OQT.
 
-Using the above-mentioned Swagger UI one can create valid `curl` requests using an interactive interface.
+### Parameters
 
-In the following sections examples for a GET and POST request for an Indicator of a region pre-defined by OQT are given.
+- `name`: Name of the Indicator
+- `layerName`: Name of the layer definition
+- `dataset`: Name of the dataset containing bounding polygons
+- `featureId`: Identifier of the feature in the dataset
+- `bpolys`: Bounding polygon(s) as GeoJSON Feature or FeatureCollection object. The Geometry of those objects has to be Polygon or MultiPolygon.
+- `layer`: Name, description and data of a custom layer. Data has to be provided in the same format as the response of the ohsome API would look like.
+- `includeSvg` (boolean): Include a SVG string of a figure displaying the Indicator
+results
+- `includeHtml` (boolean): Include a HTML string with the Indicator results
 
-### Request an Indicator for a pre-defined region
+Parameters are in part mutual exclusive (E.g. either `dataset` and `featureId` or `bpolys`. If the response contains multiple errors probably not all required parameters and/or mutual exclusive parameters has been provided.
```

Comment:
Here you miss a closing bracket. I do not fully get the second  sentence: how can a response contain "multiple errors"? In any case it should be "have been provided".

Reply:
Added the missing bracket.